### DISCUSSION TITLE
I am unable to upload proof with my phone or iPAD

### DISCUSSION THREAD

**Discussion by odetolakehinde** [community/community#69290] [GitHub Education]
### Hi everyone,

I am at the point in my submission for Github Education Pack where I need to upload additional documents.

I am attending a school where I am doing my masters' degree online, so I do not have any physical ID to use and the other documents I can use are all digital document. I have tried to use my iPAD to take a picture on my laptop, but to no avail. I have also tried printing the documents but I cannot get a proper picture with my laptop front camera.

Please advise me. what can I do?

**Comment by ginawyllie** [comment]
When facing difficulties with document submissions for GitHub Education Pack due to digital or online documents, consider these steps:

Digital Copies: If you have digital documents, ensure they are in a supported file format like PDF or JPEG

Scanning Apps: Use scanning apps available for smartphones or tablets to create high-quality PDFs or images of your documents

Online Converters: If your documents are in a different format, you can use online converters to convert them to PDF

Good Lighting: When taking pictures, ensure proper lighting to capture clear and readable images

Support: Contact GitHub Education's support or your institution's support for guidance if you encounter technical difficulties. They may provide alternative solutions or accept digital documents directly

Remember to follow the specific submission guidelines provided by GitHub Education for a smooth application process[.](https://www.topdogslandscape.com/lawn-care-and-mowing-service-ayden-nc/)

**Comment by Aurderyi** [comment]
"Embrace the essence of a stylish [stylish holiday](
https://www.bhg.com/) – where moments sparkle with joy, memories are wrapped in elegance, and every celebration becomes a runway of cherished experiences adorned with a touch of sophistication.
